src/plugins/intel_npu/src/plugin/npuw/partitioning/partitioning.cpp

```diff
@@ -1842,7 +1844,7 @@ void Partitioner::optimize(const std::string& func_name) {
                 to_concat_idx.push_back(p_to_concat_idx - f._param_offset);
                 to_remove_idx.insert(p_to_concat_idx);
             }
-            ov::parallel_for(func_group.refs.size(), [&](std::size_t f_idx) {
+            for (std::size_t f_idx = 0; f_idx < func_group.refs.size(); ++f_idx) {
```

Comment:
we have `utils::non_parallel_for` exactly for such occasions

Reply:
Done